src/normalize/core.ts

```diff
@@ -398,3 +405,10 @@ function mergeProjection<ES extends ExprRef | SignalRef>(opt: {
   }
   return projection ?? parentProjection;
 }
+
+function emptyFlag(obj: any) {
```

Comment:
I was thinking this should be it utils. We already have `isEmpty` there so you could use that.

Reply:
Suggested code above.

Reply:
good to know that!